playground/css/__tests__/css.spec.ts

```diff
@@ -284,10 +284,6 @@ test('@import dependency w/ sass entry', async () => {
   expect(await getColor('.css-dep-sass')).toBe('orange')
 })
 
-test('@import dependency w/ stylus entry', async () => {
-  expect(await getColor('.css-dep-stylus')).toBe('red')
-})
-
```

Comment:
I removed this test because this test was not testing the stylus entry and this feature wasn't working in the first place.
This test was passing because `@import 'css-dep'` was resolved by relative path.

refs #2860